yeat/cli/just_yeat_it.py

```diff
@@ -156,10 +161,14 @@ def create_config(args):
 
 
 def get_config_data(args):
+    print(args)
+    assert 0
```

Comment:
Looks like these were left in by accident

Reply:
Thanks! Good catch!